### PR TITLE
Fix #2363 per theme default avatar

### LINKED ISSUE #2363: Per theme default avatar
Make it possible to use something like `{theme}` for the default avatar as suggested here: http://community.mybb.com/thread-172906.html

### PR DESCRIPTION
**Pull request by Destroy666x**
I didn't include `$change_dir` because it doesn't seem to be implemented
correctly anyways (I'll create an issue later).

https://github.com/mybb/mybb/issues/2363

Hope it gets merged before other conflicts arise.

### CODE REVIEW

File: admin/modules/user/users.php [hunk 1]
```diff
@@ -3596,7 +3596,7 @@ function build_users_view($view)
 			}
 			if(!$user['avatar'])
 			{
-				$user['avatar'] = "../".$mybb->settings['useravatar'];
+				$user['avatar'] = "../".str_replace('{theme}', 'images', $mybb->settings['useravatar']);
```
Comment (dequeues) [inline admin/modules/user/users.php]: {theme} here isn't actually the theme, it's `images`. I'm sure that isn't intended as per title

File: admin/modules/user/users.php [hunk 2]
```diff
@@ -992,7 +992,7 @@
 	}
 	else
 	{
-		$user['avatar'] = "../".$mybb->settings['useravatar'];
+		$user['avatar'] = "../".str_replace('{theme}', 'images', $mybb->settings['useravatar']);
```
Comment (dequeues) [inline admin/modules/user/users.php]: {theme} here isn't actually the theme, it's `images`. I'm sure that isn't intended as per title

Comment (Destroy666x) [inline admin/modules/user/users.php]: That's how everything works... Check smilies, post icons, etc. You need to point to a path and `$theme` is not available in ACP.

Comment (dequeues) [inline admin/modules/user/users.php]: But that's going to show a 404 in the Users in ACP. That is going to give ../images/images/default_avatar.png

Comment (dequeues) [inline admin/modules/user/users.php]: Nevermind, I took it the wrong way

**Pull request merged**